test/WebSites/ApiExplorerWebSite/ActionDescriptorChangeProvider.cs

```diff
@@ -9,20 +9,22 @@ namespace ApiExplorerWebSite
 {
     public class ActionDescriptorChangeProvider : IActionDescriptorChangeProvider
     {
-        private ActionDescriptorChangeProvider()
+        public ActionDescriptorChangeProvider(WellKnownChangeToken changeToken)
         {
+            ChangeToken = changeToken;
         }
 
-        public static ActionDescriptorChangeProvider Instance { get; } = new ActionDescriptorChangeProvider();
```

Comment:
This really didn't work very well now that multiple things called `GetChangeToken`. Refactored the code to do this correctly now.